pulseapi/entries/models.py

```diff
@@ -73,8 +73,22 @@ class Entry(models.Model):
     """
     A pulse entry
     """
+    ENTRY_TYPES = (
+        ('base','base'),
```

Comment:
need spaces between tuple elements (this will also make travis pass)

Reply:
Ah, flake8...

In cases like these, feel free to use the "suggestion" feature for comments, and I'll just hit that "accept suggestion" button - that way you don't end up working on the PR, but code changes can still be effected without anyone having to go in and edit files.